experimental/LieAlgebras/test/Combinatorics-test.jl

```diff
@@ -1,5 +1,7 @@
 @testset "LieAlgebras.Combinatorics" begin
   @testset "combinations" begin
+    combinations = Oscar.LieAlgebras.combinations
```

Comment:
Shouldn't this be 
```suggestion
    import Oscar.LieAlgebras: combinations
```
Possibly at the top of this file? Same for the other similar ones

Reply:
Yes, this is the right way to do it. Alternatively use a `let` around everything.

Reply:
or put the tests into a module of their own?

Reply:
I learned to use `let` from the julia tests. There it is used all over the place to avoid polluting the namespace. Not sure if it makes the use of `let` better.

Reply:
P.S.: I thought `@testset` introduces automatically a local scope block.

Reply:
I am a bit confused. What would be the/your preferred way to change this into?

Reply:
Either use `import` or `let` or put everything into a module -- but don't do `combinations = Oscar.LieAlgebras.combinations`.

@thofma suggests using `let`, which seems to fit with the goal you want to achieve; I'd be happy with that as well

Reply:
I think `@testset` introduces a local scope, so the `combinations = ...` should be fine? @fingolfin can you elaborate what the problem is?